packages/rum-core/src/domain/assembly.ts

```diff
@@ -75,6 +77,7 @@ export function startRumAssembly(
   }
 
   const syntheticsContext = getSyntheticsContext()
+  const ciContext = getCiTestContext()
```

Comment:
```suggestion
  const ciTestContext = getCiTestContext()
```
for naming consistency